DepScript.py

```diff
@@ -0,0 +1,81 @@
+import os
+import platform
+import requests
+import zipfile
+import tarfile
+
+def get_os_type():
+    system = platform.system().lower()
+
+    if system == "linux":
+        return "linux"
+    elif system == "darwin":
+        return "darwin"
+    elif system == "windows":
+        return "windows"
+
+    print(f"Unsupported operating system: {system}")
+    exit(1)
+
+def download_latest_release():
```

Comment:
make this function be able to take the repo name, and repo owner
then make array of them or dict that has them, run the function over them to download all the dependencies, make sure they are downloadable the same way

@0xHunterr